dev/release/rat_exclude_files.txt

```diff
@@ -20,3 +20,4 @@
 *.sln
 .github/pull_request_template.md
 src/Apache.Arrow/Flatbuf/*
+docs/images/*
```

Comment:
I'm not sure what the rules are on which files can be excluded, but it seems unnecessary to have the licence header in an SVG?

Also, I should mention that favicon.svg was not created by me, I downloaded it from https://arrow.apache.org/visual_identity/ and just modified the height and width to fit in the header.

Reply:
No it doesn't, I reduced the height to fit the header but have kept the aspect ratio the same.

Reply:
With the docs website open beside the Arrow website, I noticed the icons looked a bit different, so I've now copied the icon from arrow-site to use as the favicon and mentioned this in the README too.